schema.ts

```diff
@@ -839,9 +839,36 @@ export interface Identity extends IdentityDefinition {
   packId: number;
 }
 
+/**
+ * An identifer for a schema property for specifying labels along with the reference to the property.
+ * This is useful for specifying a label for a property reference that uses a json path, where the
+ * label of the underlying property might not be descriptive enough at the top-level object.
+ */
+export interface PropertyIdentifierDetails {
+  label: string;
+  property: string;
+}
+
+/**
+ * An identifier for an object schema property that comprises of either an exact property match with the top-level
```

Comment:
Nit, typo in "comprises of"

Reply:
Nit ^^

Reply:
oh wait what is the typo?

Reply:
"is comprised of" is how I would read it